docs/docs/usage.md

```diff
@@ -61,6 +61,7 @@ Generates a temporary `Makefile` according to the MTA descriptor and runs the `m
 | `--strict`   | Optional  | The default value is `true`. If set to `true`, the duplicated fields and fields that are not defined in the `mta.yaml` schema are reported as errors. If set to `false`, they are reported as warnings.  | `mbt build -p=cf --strict=true`
 | BETA &nbsp;&nbsp;`-m (--mode)`   | Optional  | The possible value is `verbose`. If run with this option, the temporary `Makefile` is generated in a way that allows the parallel execution of `Make` jobs to make the build process faster.   | `mbt build -m=verbose`
 | BETA  &nbsp;&nbsp;`-j (--jobs)`   | Optional  | Used only with the `--mode` parameter. This option configures the number of `Make` jobs that can run simultaneously. If omitted or if the value is less than or equal to zero, the number of jobs is defined by the number of available CPUs (maximum 8).    | `mbt build -m=verbose -j=8`
+| BETA  &nbsp;&nbsp;`-b (--sbom-file-path)`   | Optional  | The path of SBOM file, the last part of path is file name. <br><ul><li>if sbom-file-path is null, SBOM file will not be generated  <li>sbom-file-path can be relative or abs; if relative, it is relative path to project root <li>only xml file format is supported at present, it means if file suffix is .xml, or no file suffix, xml format sbom will be generated</ul> | `mbt build --sbom-file-path sbom-gen/test.sbom.xml`
```

Comment:
The path of the SBOM file, the last part of the path is the file name. <br><ul><li>If the sbom-file-path is null, the SBOM file will not be generated.  <li>The sbom-file-path can be relative or abs; If it's relative, it is the relative path to the project root. <li>Only an XML file format is currently supported, so if the file suffix is .xml, or if there's no file suffix, an XML format SBOM will be generated.</ul> | `mbt build --sbom-file-path sbom-gen/test.sbom.xml`